dbms/src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -131,6 +138,9 @@ InterpreterSelectQuery::~InterpreterSelectQuery() = default;
 
 void InterpreterSelectQuery::init(const Names & required_result_column_names)
 {
+    /// Failpoint to make our query begin after the write action finish.
+    FAIL_POINT_PAUSE(FailPoints::pause_query_init);
```

Comment:
This comment is outdated. Maybe simply remove it:

```suggestion
    FAIL_POINT_PAUSE(FailPoints::pause_query_init);
```